pages/marketplace/new.tsx

```diff
@@ -1,13 +1,12 @@
-import { programs } from '@metaplex/js';
+import { programs, Wallet } from '@metaplex/js';
 import DomainFormItem from '@/common/components/elements/DomainFormItem';
-import { WhiteRoundedButton } from '@/components/elements/Button';
+import Button from '@/components/elements/Button';
```

Comment:
I really like the loading props for the ant button. Instead of switching to vanilla button can you look to theme the existing ant button? You can strip out all the pink stuff. We are just going with the white for primary and black for scondary with all having the full rounded corners.

Do try to tailwind but thinking something like this is the move.

https://holaplex.slack.com/archives/C02DZRTN6TZ/p1645206614278599